cm_sdk/api/cm_api.py

```diff
@@ -0,0 +1,133 @@
+from cm_sdk.client import CloudManagerClient
+from cm_sdk.models.alert_config import alert_config
+from cm_sdk.models.host import host
+from cm_sdk.models.backup_config import backup_config
+
+class Api(object):
+    """
+    MongoDB CloudManager API wrapper for CRUD operations on AlertConfigurations.
+    https://docs.cloud.mongodb.com/reference/api/alert-configurations/
+
+    Expected usage is as follows
+
+    import json
+    import pprint
+    from cm_sdk.models.alert_config import AlertConfig
+    from cm_sdk.api.alert_config import AlertConfigApi
+    import cm_sdk.client
+
+    # keys and ids have been changed to representative,
+    # but not reals, values
+    group_id = "JBAE4urukmv1GY3pukguksSZ8"
+
+    c = cm_sdk.client.CloudManagerClient(
+         "https://cloud.mongodb.com",
+         group_id,
+         "me@example.com",
+         "9705abe4-6d10-11e5-a3c6-ecf4bb625d2e")
+
+    api = AlertConfigApi(c)
+
+    fetched = api.get_alert_configs()
+    """
+    IMPLEMENTED_APIS = [
+        {"name": "AlertConfig", "status": "beta"},
+        {"name": "Alert", "status": "alpha"},
+        {"name": "BackupConfig", "status": "alpha"},
+        {"name": "Host", "status": "alpha"}]
+
+
+    def __init__(self, group_id, api_user, api_key, api_host="https://cloud.mongodb.com"):
+        self.client = CloudManagerClient(group_id, api_user, api_key, api_host)
+
+
+    def get_alert_configs(self):
+        """
+        Return all the CloudManager AlertConfig objects as a list of
+        cm_sdk.models.alert_config.AlertConfig instances.
+        """
+        to_return = []
+
+        alert_configs = self.client.get("/groups/{group_id}/alertConfigs")
+        for ac in alert_configs['results']:
```

Comment:
`return [alert_config.AlertConfig.from_dict(ac) for ac in alert_configs['results']`

Not only is that more idiomatic, it's also more performant.